script/Arcueid/Day 12/12_00_ARC12_2.txt

```diff
@@ -282,6 +282,7 @@ Still breathing heavily, I take off my glasses.
 [sha:236750aef2d02ec4e00d4a811da6f4fd70878ea2]{
 -- Page 52, Offset 30650.
 -- 懐に隠した<短剣|ナイフ>を<抜刀|ひきぬ>く。
+// is unsheathe the right word here? kinda implies the knife has a cover or something
```

Comment:
Yeah I think it's more of a switchblade, but I think it's basically the closest thing there is revert